cpp/arcticdb/entity/native_tensor.hpp

```diff
@@ -119,14 +119,16 @@ struct NativeTensor {
     [[nodiscard]] auto expanded_dim() const { return expanded_dim_; }
     template<typename T>
     const T *ptr_cast(size_t pos) const {
+        util::check(ptr != nullptr, "Unexpected null ptr in NativeTensor");
```

Comment:
Why is this no longer needed?

Reply:
It has been added. It is a bit of a sanity check because of there is a incompatibility of the numpy version used at runtime vs compile time, the whole numpy object is misread and this pointer is null